features/Extended Materials/Shaders/ExtendedMaterials/ExtendedMaterials.hlsli

```diff
@@ -311,45 +311,62 @@ namespace ExtendedMaterials
 		return coords;
 	}
 
-	// https://advances.realtimerendering.com/s2006/Tatarchuk-POM.pdf
+#if defined(TRUE_PBR)
+#	define SHADOW_INTENSITY 1.0
+#else
+#	define SHADOW_INTENSITY 2.0
+#endif
 
+	// https://advances.realtimerendering.com/s2006/Tatarchuk-POM.pdf
 	// Cheap method of creating shadows using height for a given light source
 	float GetParallaxSoftShadowMultiplier(float2 coords, float mipLevel, float3 L, float sh0, Texture2D<float4> tex, SamplerState texSampler, uint channel, float quality, float noise, DisplacementParams params)
 	{
-		[branch] if (quality > 0.0)
+		[branch] if (quality > 0.0 || extendedMaterialSettings.ExtendShadows)
 		{
-			float2 rayDir = L.xy * 0.1;
-			float4 multipliers = rcp((float4(4, 3, 2, 1) + noise));
+			float2 rayDir = L.xy * 0.1 * params.HeightScale;
```

Comment:
Moved Height Scale to multiply the ray, this resembles how the parallax function scales the steps. (viewDirTS.xy * float2(maxHeight, maxHeight) * stepSize.xx) where maxHeight = 0.1 * HeightScale